Jbarget/oxymore
# So/map test tag functionality

### What changes have you made?
- the user is now able to return to project page 
- the branch was originally created to test "tagging" on the map using css tooltips - as this is an enhancement it will not be implemented at this stage


### Which issue(s) does this PR fix?

Fixes #398 

### Screenshots (if there are design changes)


### How to test
- Set the date back to a time in the past and the content should appear
- Click the magnify icon to return to the project page
